src/org/thoughtcrime/securesms/qr/QrActivity.java

```diff
@@ -113,6 +113,13 @@ private void checkPermissions(int position, ProfilePagerAdapter adapter, ViewPag
     public boolean onPrepareOptionsMenu(Menu menu) {
       menu.clear();
       getMenuInflater().inflate(R.menu.qr_show, menu);
+      if(tabLayout.getSelectedTabPosition() == TAB_SHOW) {
+        menu.findItem(R.id.withdraw).setVisible(true);
+        menu.findItem(R.id.copy).setVisible(true);
+      } else {
```

Comment:
with `inflate()` we just added all items visible; therefore, we only need to hide them, not to make them visible again.

```suggestion
      if(tabLayout.getSelectedTabPosition() == TAB_SCAN) {
```